### ISSUE TITLE
Warning when there is no main reactor should be Information not Warning

### ISSUE THREAD

**Issue by edwardalee** [lf-lang/lingua-franca#1113]
Currently, if you have a .lf file with no main reactor, you get a warning:
```
WARNING: The given Lingua Franca program does not define a main reactor. Therefore, no code was generated.
```
This gets flagged as a yellow caution symbol in the IDE.  I don't think this should be a warning. It should be reported as "Information", but our `ErrorReporter` interface does not have support for this.  It should be straightforward to extend it, so I'm marking this as a good first issue.

**Comment by cmnrd** [comment]
I am wondering what the error reporter should do with the info message, though. Print it to the command line? I think there is no obvious way of showing info messages in an IDE. However, if info messages are always printed to the command line, we could just print them directly, right?

**Comment by edwardalee** [comment]
Eclipse has an "info" icon that it can put where it puts the warning or error icon in the left margin.

**Comment by lhstrh** [comment]
Fixed by @zekailin00 in #1192.
